src/main/java/frc/robot/commands/Arm/ArmDownCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.Arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+
+public class ArmDownCommand extends CommandBase {
+    private final ArmSubsystem armSubsystem = ArmSubsystem.getInstance();
+
+    public ArmDownCommand() {
```

Comment:
worth noting here that since subsystems are singletons, commands dont need to take them as constructor arguments. We just grab the instance as seen in line 7.